conformance/run_all.py

```
@@ -0,0 +1,276 @@
+    """A child exceeded OUTPUT_CAP_BYTES; its output is not materialized."""
+
+
+def _run_capped(cmd: list[str], cwd: Path, timeout: int) -> subprocess.CompletedProcess:
+    """subprocess.run with a ceiling on how much output is ever held.
+
+    Streams both pipes to temporary files, polls their combined size while the
+    child runs, and kills it the moment the cap is crossed. Only the first
+    OUTPUT_CAP_BYTES are ever read into memory.
+    """
+    with tempfile.TemporaryFile() as out, tempfile.TemporaryFile() as err:
+        proc = subprocess.Popen(cmd, cwd=str(cwd), stdout=out, stderr=err)
```

Ruff (0.16.1)

[error] `subprocess` call: check for execution of untrusted input

(S603)